api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -54,6 +54,32 @@ private Relationships() {
         owns
     }
 
+
+    /**
+     * The list of possible relationship (aka edges) direction. Relationships are not bidirectional.
+     */
+    public enum Direction {
+        /**
```

Comment:
For me, `incoming`, `outgoing` and `both` is clear enough. Some entity is a target of a relationship outgoing from this entity which is a source of the relationship.

Or conversely, some entity is a source of a relationship that is incoming to this entity, which is the target of the relationship.

I.e. source/target express the "position" of the entities, while incoming/outgoing express the direction of the relationship.